source/public/mvp-1/status.slim

```diff
@@ -7,18 +7,18 @@ form
   .row
     .small-12.medium-6.columns    
       br
-      h4 What is your status?
-      p If your case is against your partner, such as a divorce, dissolution or domestic violence, select single.
+      h4 Are you married or living with someone and sharing an income?
```

Comment:
The paper form asks 'What is your status?', no problem if it is different online, but I thought we found these 'yes/no' questions cause too much cognitive load.